cmd/sigclient.go

```diff
@@ -51,6 +51,38 @@ var esBulkCmd = &cobra.Command{
 	},
 }
 
+// esBulkCmd represents the es bulk ingestion
+var createLogsCmd = &cobra.Command{
```

Comment:
this whole cobra cmd is not needed, we are just going to add a new value for `-g`